package.json

```diff
@@ -14,6 +14,9 @@
     "start": "yarn storybook",
     "test": "jest",
     "test:watch": "jest --watch",
+    "lint": "tslint src/**/*.{ts,tsx} -t verbose",
```

Comment:
We should add put the glob pattern between parentheses. I have run into the same issue with ESLint before:

> If you don't quote your glob patterns, your shell can intercept them and then we can't guarantee the results. If you quote your glob patterns such that they are not intercepted by the shell, then ESLint can expand the globs in the way you expect. https://github.com/eslint/eslint/issues/9330#issuecomment-330712845

If there is no particular reason to use `verbose` format I would leave the `-t verbose` option out. So this script could look like this:

```
"lint": "tslint 'src/**/*.{ts,tsx}'",
```

Reply:
`-t verbose` is useful because it shows which TSLint rules you violate, without it, it just shows line numbers with error messages, but you don't know which rule it is.

![image](https://user-images.githubusercontent.com/9773803/56077167-39c4dc80-5dd9-11e9-9ebd-3f540d03ca40.png)

vs

![image](https://user-images.githubusercontent.com/9773803/56077170-3fbabd80-5dd9-11e9-8dad-b5abbb202601.png)

Reply:
I agree it's best to keep `-t verbose` for TSLint

Reply:
added glob between parentheses 👍